### PR TITLE
[HNC-466] updating to onerepo

### PR DESCRIPTION
**Pull request by vganpisetty**

### CODE REVIEW

File: .github/workflows/bootstrap-image.yml [hunk 1]
```diff
@@ -24,19 +24,24 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v3
 
+      - name: login to Artifactory
+        id: login
+        run: docker login ${{ env.ARTIFACTORY }} -u ${{ env.ARTIFACTORY_USER }} -p ${{ env.ARTIFACTORY_API_KEY }}
```
Comment (cardosov) [inline .github/workflows/bootstrap-image.yml:29]: Maybe keep the `docker login` as it was with the `--password-stdin` method.

File: .github/workflows/merge.yml [hunk 2]
```diff
@@ -66,22 +66,22 @@ env:
...
-  PUBLIC_SNAPSHOT_REPO_URL: ${ARTIFACTORY_BASE_URL}/pntpub-mvn-snapshot-orl
-  PRIVATE_SNAPSHOT_REPO: pntprv-mvn-snapshot-orl
+  PUBLIC_SNAPSHOT_REPO_URL: ${ARTIFACTORY_BASE_URL}/pntpub-maven-snapshot
+  PRIVATE_SNAPSHOT_REPO: pntprv-maven-snapshot
   PRIVATE_SNAPSHOT_REPO_URL: ${ARTIFACTORY_BASE_URL}/${PRIVATE_SNAPSHOT_REPO}
 
-  DOCKER_PULL_HOST: pnt-docker.repo.orl.eng.hitachivantara.com/
-  DOCKER_PUBLIC_PUSH_HOST: pntpub-docker-dev-orl.repo.orl.eng.hitachivantara.com
-  DOCKER_PRIVATE_PUSH_HOST: pntprv-docker-dev-orl.repo.orl.eng.hitachivantara.com
+  DOCKER_PULL_HOST: one.hitachivantara.com/artifactory/docker/
```
Comment (cardosov) [inline .github/workflows/merge.yml:82]: Why the `artifactory` in the URL?

Same for the next 2 lines.

File: .github/workflows/pr.yml [hunk 3]
```diff
@@ -30,22 +30,22 @@ env:
...
-  PUBLIC_SNAPSHOT_REPO_URL: ${ARTIFACTORY_BASE_URL}/pntpub-mvn-snapshot-orl
-  PRIVATE_SNAPSHOT_REPO: pntprv-mvn-snapshot-orl
+  PUBLIC_SNAPSHOT_REPO_URL: ${ARTIFACTORY_BASE_URL}/pntpub-maven-snapshot
+  PRIVATE_SNAPSHOT_REPO: pntprv-maven-snapshot
   PRIVATE_SNAPSHOT_REPO_URL: ${ARTIFACTORY_BASE_URL}/${PRIVATE_SNAPSHOT_REPO}
 
-  DOCKER_PULL_HOST: pnt-docker.repo.orl.eng.hitachivantara.com/
-  DOCKER_PUBLIC_PUSH_HOST: pntpub-docker-dev-orl.repo.orl.eng.hitachivantara.com
-  DOCKER_PRIVATE_PUSH_HOST: pntprv-docker-dev-orl.repo.orl.eng.hitachivantara.com
+  DOCKER_PULL_HOST: one.hitachivantara.com/artifactory/docker/
```
Comment (cardosov) [inline .github/workflows/pr.yml:46]: Same as `merge.yaml`

File: .github/workflows/release.yml [hunk 4]
```diff
@@ -9,7 +9,7 @@ on:
       arti_host:
         required: false
         type: string
-        default: https://repo.orl.eng.hitachivantara.com
+        default: https://one.hitachivantara.com/
```
Comment (cardosov) [inline .github/workflows/release.yml:12]: The trailing forward slash is not need.

**Review by cardosov — CHANGES_REQUESTED** [review]
This PR probably needs to be merged only after all work on Pentaho migrations is finished. Also, were there any testing builds run to check this changes?

File: .github/workflows/bootstrap-image.yml (hunk 1 above)
Comment (vganpisetty) [reply .github/workflows/bootstrap-image.yml]: updated

File: .github/workflows/pr.yml (hunk 3 above)
Comment (vganpisetty) [reply .github/workflows/pr.yml]: udated

File: .github/workflows/release.yml (hunk 4 above)
Comment (vganpisetty) [reply .github/workflows/release.yml]: removed /

File: .github/workflows/release.yml [hunk 5]
```diff
@@ -48,7 +48,7 @@ jobs:
     runs-on: [ k8s ]
 
     container:
-      image: docker.repo.orl.eng.hitachivantara.com/pentaho/actions-common:20230823.18
+      image: one.hitachivantara.com/devops-docker-release/pentaho/actions-common:20230823.18
```
Comment (cardosov) [inline .github/workflows/release.yml:51]: in [merge.yml](https://github.com/pentaho/actions-common/pull/8/files#diff-bdb37b2ad65e049a0fc043e88813fdcf7ca0118abc11d99bbda676adfcbbb5a1R94), you're referring to `actions-common:20231117.30` but here is something else, why? Do both images exist for sure?

**Review by cardosov — CHANGES_REQUESTED** [review]

File: .github/workflows/release.yml (hunk 5 above)
Comment (vganpisetty) [reply .github/workflows/release.yml]: Updated the image 
https://github.com/pentaho/actions-common/actions/runs/6903270928

**Pull request closed without merging**